pylint/checkers/imports.py

```diff
@@ -278,6 +278,11 @@ def _make_graph(filename, dep_info, sect, gtype):
         "Used when an import alias is same as original package."
         "e.g using import numpy as numpy instead of import numpy as np",
     ),
+    "C0415": (
+        "Import outside toplevel (%s)",
+        "import-outside-toplevel",
+        "Used when a module is imported from somewhere other than the toplevel",
```

Comment:
Can we adjust this wording slightly to make it more clear how to fix the message? Perhaps "Used when an import statement is used anywhere other than at the top level of the module. Move this import to the top of the file."

Reply:
fixed